radlab-ui/webapp/src/utils/terraform.ts

```diff
@@ -20,6 +21,8 @@ const GROUP_MATCHER = /group\=([\d]+)/gi
 const ORDER_MATCHER = /order\=([\d]+)/gi
 const OPTIONS_MATCHER = /options\=([^\s}]+)/gi
 const SAFE_UPDATE_MATCHER = /updatesafe/gi
+const MANDATORY_MATCHER = /mandatory/gi
+const DEPENDSON_MATCHER = /dependson\=\((.*)\)/gi
```

Comment:
```suggestion
const DEPENDSON_MATCHER = /dependson\=([a-zA-Z0-9-_=&|()<>]*)/gi
```

Reply:
Note the removal of the beginning and ending parentheses and the added ability to have parentheses in the middle of an expression:
![image](https://github.com/GoogleCloudPlatform/rad-lab/assets/69013624/c6c3353e-96b3-41c9-91c7-76c3644eeecb)